script/QA/QA_15.txt

```diff
@@ -11,152 +11,160 @@ but now I've changed my mind.
+own. She really goes off the rails when there's nobody there%{n}
 }
 [sha:7520fee54f897d6f82ad6fe56bd3b6d79d4f938b]{
 -- Page 57, Offset 43424.
 -- 彼女、ひとりになるとバランスとれなくなるので……
-It seems that her screws are loose when she's alone...
+to keep her in line...
 }
 [sha:60fd7928da0c1cb4436b3c34e1017fc4ab19d69c]{
 -- Page 58, Offset 43425.
 -- わたしもこうならないよう注意していたのですが、
-I have been careful to not let this happen so far,
+I'd been careful about not letting this happen so far, but%{n}
 }
 [sha:3b0eb7506a7dc57cec194160014cd2e59b0a9602]{
 -- Page 58, Offset 43426.
 -- 諸事情あって目を離してしまったので……
-but this time I slipped up for various reasons...
+// Ref to the fact you can enter this dead end 2 ways
+there are a couple reasons I could have been forced to%{n}
+take my eyes off her...
 }
 [sha:6d6e667d86c10f8bf4ff72e08fa7527828eead4b]{
 -- Page 59, Offset 43427.
 -- ふーん。
-Hmm.
+Hmm.%{n}
 }
 [sha:a5fb138cb8b6c31f0f6e233a128d563cc4e31f71]{
 -- Page 59, Offset 43428.
 -- 諸事情って、どんな？
-And what were those reasons exactly?
+What kind of reasons?
 }
 [sha:7ba823cf809294b828c21e3b0d335b3401e8c9cf]{
 -- Page 60, Offset 43429.
 -- ａ．シスター・ノエルの密告を聞いた
-(a) The Priest Executor summoned Executor Ciel
+Reason A: Following a tip from Sister Noel, the acting%{n}
 }
 [sha:c442a7688db44a36751b041ad2db509005a87fb0]{
 -- Page 60, Offset 43430.
 -- 司祭代行が代行者シエルを招集してしまった。
-after hearing an anonymous report from Sister Noel.
+presbyter summons for Ciel.
 }
 [sha:43cc10e226334541a788446677e6d5bef3b83e3a]{
 -- Page 61, Offset 43431.
 -- ｂ．単独で下水道に向かった代行者シエルは
-(b) Executor Ciel, who went into the sewers alone, learned of
+Reason B: Executor Ciel went down into the sewers alone,%{n}
 }
 [sha:bbb3a625bfc5a9d902b2623b09b13dd5c7d2c125]{
 -- Page 61, Offset 43432.
 -- ロアの状態を知り、遠野邸に捜索に向かった。
-Roa's condition and went to Tohno's mansion to search for answers.
+learned what happened to Roa, and went to the Tohno%{n}
+mansion in search of answers.
 }
 [sha:7ee8f38b956129a41883343efc895a1348a22c6d]{
 -- Page 62, Offset 43433.
 -- のどちらかですね。
-It could be one or the other.
+%{n}Either of these are possible.
 }
 [sha:f923250e7d1952eb0454ad1b0257ffebc3da37da]{
 -- Page 63, Offset 43434.
 -- ａの方は午後に招集の報せが届いたので、
-In terms of (a), it would have been good if I was in a
+In case A, she received her summons during the afternoon,%{n}
 }
 [sha:02402beb2841ff0433d0cc54b23489ab011b0f34]{
 -- Page 63, Offset 43435.
 -- それを無視できるような状態であれば良かった。
-situation where I could ignore the call to summons in the afternoon.
+so perhaps it would be better if she had some reason she%{n}
+could ignore it.
 }
 [sha:1e08174e90077a0bf3b4968f0245f5d7f94fd21f]{
 -- Page 64, Offset 43436.
 -- ｂの方は言わずもがなです。
-I don't have to tell you about the case of (b).
+Case B doesn't require much explaining.%{n}
 }
 [sha:b9f5771163a7da60af6731ebce3d7a64363f8196]{
 -- Page 64, Offset 43437.
 -- 遠野くんの遠慮が悪い結果を招いてしまった。
-It was due to Tohno-kun's restraint that led to a bad result.
+Tohno-kun's hesitance is what led to a bad outcome.
```

Comment:
I imagine space is tight? But it reads very strangely. Maybe could be shifted wording a bit.

>Tohno-kun's restraint had bad consequences.